src/index.js

```diff
@@ -27,6 +27,8 @@ program
   .description('Electronegativity is a tool to identify misconfigurations and security anti-patterns in Electron applications.')
   .option('-i, --input <path>', 'input [directory | .js | .html | .asar]')
   .option('-c, --checks <checkNames>', 'only run the specified checks, passed in csv format')
+  .option('-S, --severity <severitySet>', 'only return findings above the passed level of severity')
```

Comment:
What about having:
* `-c` alias of `--confidence`
* `-s` alias of `--severity`
* `-l` (previously `-c`) alias `--checks`?

Reply:
I think that's a reasonable change. I moved `-c` to `-l` in ab71ef4727fc9eaf3b26de70399c6bbc66d26326.
Since it's a minor change we can revert that easily.